src/detections/hooks/use-split.js

```diff
@@ -0,0 +1,7 @@
+import { useLocation } from 'react-router-dom'
+
+export const useSplit = () => {
+  const link = useLocation()
+
+  return link.search ? link.search.split('=')[1].split('&')[0] : null
```

Comment:
Also, we can return an array directly from this hook. Otherwise, when the URL contains no query string, the server will receive an array containing a single element: `null`.

Reply:
Done